python/ray/data/_internal/logical/operators/read_operator.py

```diff
@@ -14,8 +14,7 @@ def __init__(
         ray_remote_args: Dict[str, Any] = None,
         read_args: Dict[str, Any] = None,
     ):
-        super().__init__("Read", [])
+        super().__init__("Read", None, ray_remote_args)
```

Comment:
Nit: Could just supply `ray_remote_args` as a kwarg and let the `input_op` default in `AbstractMap` do its thing.
```suggestion
        super().__init__("Read", ray_remote_args=ray_remote_args)
```